drivers/cc110x/gnrc_cc110x.c

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (C) 2016-17 Freie Universität Berlin
+ *               2018 Otto-von-Guericke-Universität Magdeburg
```

Comment:
Why didn't you kept it a seperate module btw?

Reply:
For no particular reason, to be honest. I just so `xbee` didn't use a separate module so I thought that's the way to do it.

@AnonMall is working on a CC1200 driver here https://github.com/RIOT-OS/RIOT/pull/7598 and I believe this piece of code could be shared between both drivers. Any suggestion where to best put the module? (Also renaming it and also renaming `NETDEV_TYPE_CC110X` would be a good preparation for sharing it with the CC1200 driver.)

Reply:
When in doubt, and more than one driver is using it I'd say `sys/net/gnrc/netif/cc1xxx` => `gnrc_netif_cc1xxx` (or however you like to call your submodule then).

Reply:
I'm not sure if a name not tied to specific transceivers would be better. Basically it uses this frame format:

```
 0                   1                   2                   3
 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
|  Destination  |    Source     |  Payload...
+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

So any transceiver that uses layer 2 addresses which are 1 byte in size and wants to carry 6LoWPAN should be able to use this. (Currently layer 2 address `0x00` is used for broadcast, which is how CC110x and CC1200 do it, but that could easily be generalized. I assume there will only be three cases used in practice: `0x00` as broadcast, `0xFF` as broadcast and no broadcast at all.)

Should I split that off as a separate PR? I think this could be reviewed and merged quite fast. Then I could adapt this PR to use that. If so: Should I generalize the broadcast right away? (Maybe other uses than CC110x and CC1200 never appear, so this generalization would be wasted effort.)

I'm really bad at giving names. Any suggestion would be welcome :-)

Reply:
Fixed with https://github.com/RIOT-OS/RIOT/pull/10638